src/rpc/util.cpp

```diff
@@ -733,3 +733,21 @@ std::vector<CScript> EvalDescriptorStringOrObject(const UniValue& scanobject, Fl
     }
     return ret;
 }
+
+UniValue GetServicesNames(ServiceFlags services)
```

Comment:
Should also `#include <protocol.h>`

Reply:
It is included. https://github.com/bitcoin/bitcoin/pull/16787/files